Dockerfile

```diff
@@ -35,14 +37,12 @@ ENV PYTHONDONTWRITEBYTECODE=true
 WORKDIR /home/conda/
 
 COPY network.h5 ${WORKDIR}network.h5
+COPY conda-env.yml ${WORKDIR}conda-env.yaml
 
 ARG S3_PYPI_HOST
 ARG SDIST_SPEC
 
-RUN conda create -n hyp3-water-mask -c conda-forge python=3.7 boto3 gdal imageio importlib_metadata \
-    keras lxml matplotlib netCDF4 numpy pillow proj \
-    psycopg2 pyshp pytest pytest-console-scripts pytest-cov requests scipy \
-    setuptools six statsmodels wheel && \
+RUN conda env create -f conda-env.yml && \
```

Comment:
@Alex-Lewandowski note that we can now use a conda environment YAML file to setup the environment for both the docker image, the CI/CD jobs, and on your local machine!